stdlib/Distributed/src/remotecall.jl

```diff
@@ -177,9 +185,10 @@ If the argument `Future` is owned by a different node, this call will block to w
 It is recommended to wait for `rr` in a separate task instead
 or to use a local [`Channel`](@ref) as a proxy:
 
-    c = Channel(1)
-    @async put!(c, remotecall_fetch(long_computation, p))
-    isready(c)  # will not block
+    p = 1
```

Comment:
Maybe make this a proper
````
```julia
```
````
while you are at it?

Reply:
@fredrikekre no indentation required in that case right?

Reply:
No.